Tests/RegressionTests.cs

```diff
@@ -705,19 +705,19 @@ private static TestCaseData[] GetRegressionTestParameters()
                 {"Compounding Annual Return", "-100.000%"},
                 {"Drawdown", "5.400%"},
                 {"Expectancy", "-1"},
-                {"Net Profit", "-5.656%"},
-                {"Sharpe Ratio", "-20.372"},
+                {"Net Profit", "-5.603%"},
+                {"Sharpe Ratio", "-19.82"},
                 {"Loss Rate", "100%"},
                 {"Win Rate", "0%"},
                 {"Profit-Loss Ratio", "0"},
                 {"Alpha", "-11.165"},
-                {"Beta", "574.85"},
-                {"Annual Standard Deviation", "0.353"},
-                {"Annual Variance", "0.125"},
-                {"Information Ratio", "-20.427"},
-                {"Tracking Error", "0.353"},
-                {"Treynor Ratio", "-0.013"},
-                {"Total Fees", "$6076.08"}
+                {"Beta", "585.081"},
```

Comment:
I'd be curious how these specifically changed. For example, were we previously filling using fill-forward trade data? Or perhaps we were filling using trade data and now it's filling using quote data? Changes in regression can be expected with bugfixes but I think it's critically important to be able to completely explain *why* the expected results have changed.